azure-pipelines.sds.pr-release.yml

```diff
@@ -61,7 +61,7 @@ stages:
                 - "[*]SchedulerJobs.Common.*"
                 - "[*]SchedulerJobs.UnitTests.*"
                 - "[Testing.Common]*"
-
+               
```

Comment:
you need to add this line:

`dotnetCoreVersion: "6.x"`

to the Unit Tests parameters section.

You can use notification API as a reference: https://github.com/hmcts/vh-notification-api/blob/master/azure-pipelines.sds.pr-release.yml#L62

Reply:
this hasn't been done yet